docs/unraid-os/system-administration/secure-your-server/tailscale.md

```
@@ -0,0 +1,184 @@
+---
+sidebar_position: 6
+sidebar_label: Tailscale
+---
+
+import Tabs from '@theme/Tabs';
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)